mp/src/materialsystem/stdshaders/pbr_dx9.cpp

```diff
@@ -271,24 +290,52 @@ BEGIN_VS_SHADER(PBR, "PBR shader")
 			else
 			{
 				// We need the position, surface normal, and vertex compression format
-                unsigned int flags = VERTEX_POSITION | VERTEX_NORMAL;
+				unsigned int flags = VERTEX_POSITION | VERTEX_NORMAL | VERTEX_TANGENT_S | VERTEX_TANGENT_T;
```

Comment:
This is only needed for the ps2.0 version right?

Reply:
this is needed for parallax occlusion.